### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by tjkohli** [luwes/lite-vimeo-embed#8]
The `<button>` that is generated should have some sort of `aria-label`. I suppose this should be pretty easy. I've suggested an improvement.
